src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayWithClones = [];
+  let currentState = { ...state };
+
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (type) {
+      case 'addProperties':
```

Comment:
Move to the object. Example of usage: `case Types.AddProperties:`

Reply:
Sorry, cannot get it. Is there a link with a description of this method? As I know how the classic 'switch case' works only. Thank you